projects/arrays/text-processor/src/index-alternate.test.ts

```diff
@@ -0,0 +1,126 @@
+import { describe, expect, test } from "@jest/globals";
+import { expectType } from "tsd";
+
+import * as index from "./index";
+import * as alternate from "./solution-alternate";
+
+const { alignTexts } = process.env.TEST_SOLUTIONS ? alternate : index;
```

Comment:
CI scripts were renaming `solution_alternate` to `index_alternate`, so I just renamed it to `alternate`.